src/util.cpp

```diff
@@ -1202,7 +1202,42 @@ CInstPictures::EInstCategory CInstPictures::GetCategory ( const int iInstrument
 }
 
 // Locale management class -----------------------------------------------------
-QString CLocale::GetCountryFlagIconsResourceReference ( const QLocale::Country eCountry )
+QLocale::Country CLocale::WireFormatCountryCodeToQtCountry ( unsigned short iCountryCode )
+{
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
+    // The Jamulus protocol wire format gives us Qt5 country IDs.
+    // Qt6 changed those IDs, so we have to convert back:
+    return (QLocale::Country) wireFormatToQt6Table[iCountryCode];
+#else
+    return (QLocale::Country) iCountryCode;
+#endif
+}
+
+unsigned short CLocale::QtCountryToWireFormatCountryCode ( const QLocale::Country eCountry )
+{
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
+    // The Jamulus protocol wire format expects Qt5 country IDs.
+    // Qt6 changed those IDs, so we have to convert back:
+    return qt6CountryToWireFormat[(unsigned short) eCountry];
+#else
+    return (unsigned short) eCountry;
+#endif
+}
+
+bool CLocale::IsCountryCodeSupported ( unsigned short iCountryCode )
+{
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
+    // On newer Qt versions there might be codes which do not have a Qt5 equivalent.
+    // We have no way to support those sanely right now.
```

Comment:
Should be a // TODO: comment for future versions (if there's a workaround found)

Reply:
Hrm, the comment just intends to explain why we do a sanity check here. We don't know if and in what way this might become an issue, so there's no specific TODO right now, I'd say?